spec/knapsack/adapters/rspec_adapter_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'ostruct'
```

Comment:
Why do we need this? Was something failing because ostruct was not loaded?

Reply:
Tests for Ruby v2.3 [started failing](https://github.com/KnapsackPro/knapsack/actions/runs/9401495650/job/25893400671).

I didn't investigate it much, but maybe a change in a patch version (2.3.x)?